code/renderers/react/package.json

```diff
@@ -31,6 +31,11 @@
       "import": "./dist/config.mjs",
       "types": "./dist/config.d.ts"
     },
+    "./dist/preset": {
```

Comment:
Same as above, this should just be `"./preset"`

Reply:
I don't think so!